### PR TITLE
Add glossary entries for static and mirror pods

### PR DESCRIPTION
**Pull request by sftim**
Associated with #5993

### CODE REVIEW

**Comment by sftim** [comment]
I wasn't sure about tagging these entries as `fundamental`. Right now, static pods are a fairly key concept. I know there's discussion about dropping static pods from `kubelet`.

Is there a tag that's more appropriate than `fundamental`?

**Comment by DanyC97** [comment]
@sftim maybe you want to leave a comment on the main issue to let people know.

Would have been nice if the bullet list in the main Issue would have been updated so people know what is left to be completed rather than going through the code.

**Comment by sftim** [comment]
The main issue has an automatic reference:
> @sftim sftim referenced this issue 4 days ago
> Add glossary entries for static and mirror pods #12684 

I'd not added a comment, I suspect that would be repeating what GitHub has already said.
If we had a magic wand, Prow could take care of ticking the bullets too.

File: content/en/docs/reference/glossary/static-pod.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Static Pod
+id: static-pod
+date: 2091-02-12
+full_link: /docs/tasks/administer-cluster/static-pod/
+short_description: >
+  A pod configured directly into Kubelet on a specific node
```
Comment (zparnold) [inline content/en/docs/reference/glossary/static-pod.md]: I don't think "into" is the right preposition here. Probably "by the" makes more sense.

File: content/en/docs/reference/glossary/mirror-pod.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+aka: 
+tags:
+- fundamental
+---
+ A {{< glossary_tooltip text="pod" term_id="pod" >}} object that a Kubelet uses
+ to represent a {{< glossary_tooltip text="static pod" term_id="static-pod" >}}
+
+<!--more--> 
+
+When Kubelet finds a static pod in its configuration, it automatically tries to
```
Comment (zparnold) [inline content/en/docs/reference/glossary/mirror-pod.md]:
```suggestion
When the Kubelet finds a static pod in its configuration, it automatically tries to
```

**Comment by zparnold** [comment]
@sftim Made some comments 😄

File: content/en/docs/reference/glossary/static-pod.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/reference/glossary/static-pod.md]: If I configure a static pod by editing the appropriate Kubelet configuration file, it's me that's doing the configuring and Kubelet that's being configured.
I want to summarize that relationship.

I'll have a think about wording here.

Comment (Bradamant3) [reply content/en/docs/reference/glossary/static-pod.md]: the kubelet (note capitalization! it's lowercase, please change throughout) is an agent that manages pods by means of specs/config files. "into" suggests that the kubelet contains the pod, which it doesn't. The kubelet is configured to manage the pod (although that's the wrong emphasis here, too). Try reusing more of the language from the task topic -- it's pretty solid.

**Comment by sftim** [comment]
@Bradamant3: do you see anything that I missed?

**Comment by Bradamant3** [comment]
@sftim apologies for the delay!

/lgtm
/approve

**Pull request merged**